packages/web/src/store/reducers.ts

```diff
@@ -47,10 +47,14 @@ import player from 'store/player/slice'
 import playlistLibrary from 'store/playlist-library/slice'
 import reachability from 'store/reachability/reducer'
 
+import { webStoreContext } from './storeContext'
+
+export const commonStoreReducers = clientStoreReducers(webStoreContext)
```

Comment:
The crux of this PR is that common store reducers (and sagas below) are initialized with a context that has local storage handlers (and could have other things).